src/views/Dashboard.vue

```diff
@@ -316,9 +302,10 @@ export default {
       if (this.isBComp()) {
         this.checkPendingFilings(this.historyFilings)
       }
-      // check for completed filing on an incoporation application
+      // Set completed filing required to show address since this is an NR
+      // once transitioned to the business id, addresses/directors will be available
       if (this.nrNumber) {
-        this.checkCompletedIncorporationFilings(this.historyFilings)
+        this.completedFilingRequired = true
```

Comment:
Will this get set if this.historyFilings doesn't change (ie, there aren't any filings)?

Reply:
It is being triggered on my local build, so it looks like it does.

Reply:
Perhaps I should move it to mounted() ?

Reply:
Sure, if you want to keep using that property. Or use the nrNumber check directly where needed?

Either is OK with me.